handlers.go

```diff
@@ -390,11 +419,22 @@ func (b *BouncerHandler) shouldAttribute(reqParams *BouncerParams) bool {
 		}
 	}
 
+	// Check if the request is coming from RTAMO, and if so, only attribute
+	// if there is a referer header from https://www.mozilla.org/
+	// https://github.com/mozilla-services/go-bouncer/issues/347
+	if fromRTAMO(reqParams.AttributionCode) {
+		if reqParams.Referer != "https://www.mozilla.org/" {
```

Comment:
What about cases where the referer header includes a path?

Reply:
Added a regex to match anything starting with `https://www.mozilla.org/`